test/upgrade/prober/wathola/sender/services.go

```diff
@@ -104,15 +116,52 @@ func NewCloudEvent(data interface{}, typ string) cloudevents.Event {
 	return e
 }
 
+// ResetEventSenders will reset configured event senders to defaults.
+func ResetEventSenders() {
+	eventSenders = make([]EventSender, 0, 1)
```

Comment:
`eventSenders = nil` will work as well if you use `var eventSenders []EventSender` to declare the variable at the top.

Reply:
Same answer as https://github.com/knative/eventing/pull/4815#discussion_r635174556